docs/content/docs/how-to/writing-tables.md

```diff
@@ -268,3 +268,146 @@ For more information of 'delete', see
 {{< /tab >}}
 
 {{< /tabs >}}
+
+## Merging into table
+
+Table Store supports "MERGE INTO" via submitting the 'merge-into' job through `flink run`. The design referenced such syntax:
```

Comment:
Document only primary key table is supported, you can refer to https://nightlies.apache.org/flink/flink-table-store-docs-master/docs/concepts/primary-key-table/